CHANGELOG.md

```diff
@@ -1,7 +1,30 @@
 Magento Functional Testing Framework Changelog
 ================================================
-3.0.0
------
+3.0.0 RC1
+---------
+
+### Enhancements
+* Customizability
+    * Introduced MFTF helpers `<helper>` to create custom actions outside of MFTF.
+    * Removed deprecated actions `<executeSelenium>` and `<performOn>`.
+* Maintainability
+    * Schema updates for test entities to only allow single entity per file.
+    * Removed support to read test entities from `<magento>dev/tests/acceptance/tests/functional/Magento/FunctionalTest`. 
```

Comment:
Please also add \<module file=""/`> is removed from Suite schema

Reply:
done